src/vl-map.js

```diff
@@ -57,16 +57,25 @@ export class VlMap extends VlElement(HTMLElement) {
         return this._shadow.querySelector('#map');
     }
 
-    get _projection() {
-        return new ol.proj.Projection({
-            code: 'EPSG:31370',
-            extent: this._extent
-        });
-    }
-
-    get _extent() {
-        return [9928, 66928, 272072, 329072];
-    }
+  get _porjectionCode() {
```

Comment:
Typo porjectionCode naar _projectionCode

Reply:
Ook zou ik dit public maken, aangezien we deze zouden nodig hebben.

Reply:
Indien akkoord met voorstel, kan dit privaat bijven.